src/NuGet.Services.Metadata.Catalog.Monitoring/Utility/CommonLogger.cs

```diff
@@ -11,12 +13,15 @@ namespace NuGet.Services.Metadata.Catalog.Monitoring
     /// </summary>
     public class CommonLogger : Common.ILogger
     {
-        public CommonLogger(ILogger logger)
+        private const int DefaultLogEventId = 23847;
```

Comment:
It might be good to add a comment to that effect.  "This event ID is believed to be unused anywhere else but is otherwise arbitrary."